cmd/ecloud/ecloud_vpnendpoint.go

```diff
@@ -0,0 +1,235 @@
+package ecloud
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/ukfast/cli/internal/pkg/factory"
+	"github.com/ukfast/cli/internal/pkg/helper"
+	"github.com/ukfast/cli/internal/pkg/output"
+	"github.com/ukfast/sdk-go/pkg/service/ecloud"
+)
+
+func ecloudVPNEndpointRootCmd(f factory.ClientFactory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "vpnendpoint",
+		Short: "sub-commands relating to VPN endpoints",
+	}
+
+	// Child commands
+	cmd.AddCommand(ecloudVPNEndpointListCmd(f))
+	cmd.AddCommand(ecloudVPNEndpointShowCmd(f))
+	cmd.AddCommand(ecloudVPNEndpointCreateCmd(f))
+	cmd.AddCommand(ecloudVPNEndpointUpdateCmd(f))
+	cmd.AddCommand(ecloudVPNEndpointDeleteCmd(f))
+
+	return cmd
+}
+
+func ecloudVPNEndpointListCmd(f factory.ClientFactory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "list",
+		Short:   "Lists VPN endpoints",
+		Long:    "This command lists VPN endpoints",
+		Example: "ukfast ecloud vpnendpoint list",
+		RunE:    ecloudCobraRunEFunc(f, ecloudVPNEndpointList),
+	}
+
+	cmd.Flags().String("name", "", "VPN endpoint name for filtering")
+
+	return cmd
+}
+
+func ecloudVPNEndpointList(service ecloud.ECloudService, cmd *cobra.Command, args []string) error {
+	params, err := helper.GetAPIRequestParametersFromFlags(cmd,
+		helper.NewStringFilterFlagOption("name", "name"),
+	)
+	if err != nil {
+		return err
+	}
+
+	endpoints, err := service.GetVPNEndpoints(params)
+	if err != nil {
+		return fmt.Errorf("Error retrieving VPN endpoints: %s", err)
+	}
+
+	return output.CommandOutput(cmd, OutputECloudVPNEndpointsProvider(endpoints))
+}
+
+func ecloudVPNEndpointShowCmd(f factory.ClientFactory) *cobra.Command {
+	return &cobra.Command{
+		Use:     "show <endpoint: id>...",
+		Short:   "Shows a VPN endpoint",
+		Long:    "This command shows one or more VPN endpoints",
+		Example: "ukfast ecloud vpnendpoint show vpns-abcdef12",
```

Comment:
Examples should use prefix `vpne-` for vpn endpoints

Reply:
Good spot, sorted :)